### DISCUSSION TITLE
使用cuda 11.3时，识别内容有问题，看了官方文档cuda支持列表里面没有cuda11.3.请问一下，这个可以支持一下吗？机器上的环境是cuda11.3，没法更改

### DISCUSSION THREAD

**Discussion by Zauther** [PaddlePaddle/PaddleOCR#12911] [Q&A] [answered]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by Sencc** [comment, marked as answer]
我也是11.3，识别没问题呀，这个不影响的
